.env.test

```
@@ -4,7 +4,7 @@
 GETH_DOCKER_IMAGE=ethereum/client-go:v1.13.14
 # Use either image or local binary for the testing
 GETH_BINARY_DIR=
-LIGHTHOUSE_DOCKER_IMAGE=sigp/lighthouse:latest-amd64-unstable
+LIGHTHOUSE_DOCKER_IMAGE=sigp/lighthouse:latest
```

```suggestion
LIGHTHOUSE_DOCKER_IMAGE=sigp/lighthouse:v7.0.1
```

might be best to just pin to a specific release